app/scripts/plugins/counterfactualMiddleware.js

```diff
@@ -17,6 +17,9 @@ function createCounterfactualMiddleware (counterfactualController, metamaskContr
           case 'counterfactual:request:balances':
             res.result = await counterfactualController.metamaskRequestBalancesRPC(req.params[0], req.params[1])
             break
+          case 'counterfactual:request:indexed_balances':
```

Comment:
What about using a more meaningful name for this? `request:indexedBalances`? (Also, we're trying to use camelCase everywhere).

Reply:
I just followed the example from this other ones:
`counterfactual:request:deposit_start`
`counterfactual:request:deposit_confirmed`

I could replace it with `balances_token_indexed`? The only other one that's camelCase it's `createChannel`